test/files/jvm/scala-concurrent-tck.scala

```diff
@@ -1018,7 +1018,3 @@ with Exceptions
 with GlobalExecutionContext
 with CustomExecutionContext
 with ExecutionContextPrepare
-{
-  System.exit(0)
```

Comment:
on 2.13 there's a copy of this test that will need adjusting. couldn't spot any other on 2.13.